io.github.clash_for_windows.appdata.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>io.github.clash_for_windows</id>
+  <launchable type="desktop-id">io.github.clash_for_windows.desktop</launchable>
+  <name>Clash for Windows</name>
+  <developer_name>Fndroid</developer_name>
+  <summary>A proxy tool.</summary>
+  <metadata_license>CC0-1.0</metadata_license>
```

Comment:
it's missing a project license.

Reply:
Because of the special nature of this software and the special policy of the Chinese government, this software is not open source now, I will explain it in detail in a new pr later, thanks again.